crm_helpdesk/views/crm_helpdesk_menu.xml

```diff
@@ -46,7 +46,7 @@
 
         <menuitem name="Helpdesk and Support" id="menu_help_support_main"
             groups="sales_team.group_sale_salesman"
-            parent="sales_team.menu_sales" action="crm_case_helpdesk_act111"/>
+            parent="sale.sale_order_menu" action="crm_case_helpdesk_act111"/>
```

Comment:
The same.

Reply:
@pedrobaeza , can you review this PR? I think that its all right ;)
Thanks!